wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+            string_fastq,
+        ])
+
+        Array[File] sample_fastqfile = s_fastq
+    }
+    Array[File] original_fastqfiles = flatten(select_all([
+        sample_srafile,
+        sample_fastqfile,
+    ]))
+
+    ### ------------------------------------------------- ###
+    ### ---------------- S E C T I O N 1 ---------------- ###
+    ### ----------- B: remove Spike-IN reads ------------ ###
+    ### ------------------------------------------------- ###
+
+    # if multiple fastqfiles are provided
+    Boolean multi_fastq = if length(original_fastqfiles) > 1 then true else false
+    Boolean one_fastq = if length(original_fastqfiles) == 1 then true else false
+
+    if (defined(spikein_bowtie_index) || defined(spikein_reference)) {
+        scatter (eachfastq in original_fastqfiles) {
+            call fastqc.fastqc as spikein_indv_fastqc { input:
+                inputfile = eachfastq,
+                default_location = if (one_fastq) then sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/SpikeIn/FastQC" else "SAMPLE/" + sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/SpikeIn/FastQC",
+            }
+            call util.basicfastqstats as spikein_indv_bfs { input:
+                fastqfile = eachfastq,
+                default_location = if (one_fastq) then sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/SpikeIn/SummaryStats" else "SAMPLE/" + sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/SpikeIn/SummaryStats",
+            }
+            call bowtie.spikein_SE as spikein_indv_map { input:
+                fastqfile = eachfastq,
+                index_files = actual_spikein_bowtie_index,
+                metricsfile = spikein_indv_bfs.metrics_out,
+                default_location = if (one_fastq) then sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/SpikeIn/SummaryStats" else "SAMPLE/" + sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/SpikeIn/SummaryStats",
+            }
+        }
+
+        Array[File] spikein_fastqfiles = spikein_indv_map.unaligned
+    }
+    Array[File] fastqfiles = select_first([
+        spikein_fastqfiles,
+        original_fastqfiles,
+    ])
+
+    ### ------------------------------------------------- ###
+    ### ---------------- S E C T I O N 2 ---------------- ###
+    ### ---- A: analysis if multiple FASTQs provided ---- ###
+    ### ------------------------------------------------- ###
+
+    if (multi_fastq) {
+        scatter (eachfastq in fastqfiles) {
+            # Execute analysis on each fastq file provided
+            # Analysis executed:
+            #   FastQC
+            #   FASTQ read length distribution
+            #   Reference Alignment using Bowtie (-k2 -m2)
+            #   Convert SAM to BAM
+            #   FastQC on BAM files
+            #   Remove Blacklists (if provided)
+            #   Remove read duplicates
+            #   Summary statistics on FASTQs
+            #   Combine html files into one for easy viewing
+
```

Comment:
This blank too.

Reply:
Similar issue as directly above. IDK if I can come up with logic to delete this one that wouldn't also delete "desirable" blanks.